app/org/sagebionetworks/bridge/json/ScheduleDeserializer.java

```diff
@@ -20,7 +23,7 @@
 import com.fasterxml.jackson.databind.DeserializationContext;
 import com.fasterxml.jackson.databind.JsonDeserializer;
 import com.fasterxml.jackson.databind.JsonNode;
-import com.google.common.collect.Lists;
+import com.fasterxml.jackson.databind.node.ArrayNode;
 
 public class ScheduleDeserializer extends JsonDeserializer<Schedule> {
```

Comment:
I think you should be able to use a serializer for just the local time list without having to create a serializer for the whole class.